server/service.go

```diff
@@ -304,6 +310,14 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 	m.bids[bidKey(slot, result.bidInfo.blockHash)] = result
 	m.bidsLock.Unlock()
 
+	// Decide response content type (JSON by default)
+	clientAccepts := goacceptheaders.Parse(req.Header.Get("Accept"))
```

Comment:
since this function deals with many clients and ends, better names would add clarity (i.e. proposer and relay accept headers)

```suggestion
	relayAcceptsContentTypes := goacceptheaders.Parse(req.Header.Get("Accept"))
```